### PR TITLE
8133 Contact Type Refactoring

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

**Review by sutt0n — APPROVED** [review]

**Review by rachelschneiderman — COMMENTED** [review]
Is a migration script needed for this refactor?

**Comment by sutt0n** [comment]
@rachelschneiderman:
> Is a migration script needed for this refactor?

Yes! I labeled this as `Don't Merge` until that happens. Good catch!

File: shared/src/business/useCases/validateAddPetitionerInteractor.js [hunk 1]
```diff
@@ -15,6 +15,8 @@ exports.validateAddPetitionerInteractor = (applicationContext, { contact }) => {
     applicationContext,
   }).getFormattedValidationErrors();
 
+  console.log('petitionerErrors', petitionerErrors);
```
Comment (kkoskelin) [inline shared/src/business/useCases/validateAddPetitionerInteractor.js]: Do we want to remove this console.log ?

File: web-client/src/presenter/actions/setDefaultCaseDetailTabAction.test.js [hunk 2]
```diff
@@ -25,6 +25,25 @@ describe('setDefaultCaseDetailTabAction', () => {
     );
   });
 
+  it('should set the partyViewTab based on the props.ppartiesTab provided', async () => {
```
Comment (kkoskelin) [inline web-client/src/presenter/actions/setDefaultCaseDetailTabAction.test.js]: _nit_ spelling `props.ppartiesTab`

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**